en/documentation/faq/4/index.md

```diff
@@ -361,19 +361,20 @@ the method is invoked. It is evaluated using the scope of the method.
 
 ### How do I pass arguments to a block?
 
+Typically, arguments are passed to a block using `yield` (or an iterator
+that calls `yield`), or by using the `Proc.call` method.
```

Comment:
I had previously just moved this paragraph to the top, but it should also be altered to use the appropriate syntax to indicate an instance method on Proc.  I prefer the `#` syntax, but this should use whichever is most consistent with other parts of this FAQ.

```suggestion
that calls `yield`), or by using the `proc.call` method.
```

```suggestion
that calls `yield`), or by using the `Proc#call` method.
```